plugins/inputs/exec/exec.go

```diff
@@ -39,12 +42,13 @@ const sampleConfig = `
   data_format = "influx"
 `
 
-const MaxStderrBytes = 512
+const MaxStderrBytes int = 512
```

Comment:
What happens if the user specifies more than this max? You maybe should check that in `Init()`, clip to max and issue a warning...